example-get-started/generate.sh

```diff
@@ -42,7 +42,7 @@ git tag -a "1-dvc-init" -m "DVC initialized."
 mkdir data
 dvc get https://github.com/iterative/dataset-registry \
         get-started/data.xml -o data/data.xml
-dvc add data/data.xml --desc "Initial XML StackOverflow dataset (raw data)"
```

Comment:
May we can actually keep both? Change the message a bit? I hope it will be visible in the viewer then.

Reply:
Sure, I kept the old desc here and changed the import desc.